### ISSUE TITLE
Some of the transactions posted through Bundlr are not returned by the arweave.net GQL endpoint

### ISSUE THREAD

**Issue by ppedziwiatr** [ar-io/arweave-gateway#3]
Example:
txId: `Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg`
Works on Viewblock: https://viewblock.io/arweave/tx/Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg
Works via `arweave.net/{txId}`: https://arweave.net/Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg

but does not work via GQL (i.e. it cannot be found): 
<img width="760" alt="image" src="https://user-images.githubusercontent.com/84311757/195123930-220a832d-eb24-4ae1-ae51-959889bb0738.png">

The gql query as curl:
```
curl 'https://arweave.net/graphql' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: https://arweave.net' --data-binary '{"query":"query {\n  transactions(\n    ids: [\n      \"Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg\"\n    ]\n  ) {\n    edges {\n      node {\n          id\n          tags {\n            name\n            value\n          }\n          block {\n            height\n          }\n        \tbundledIn {\n            id\n          }\n      \n        }\n    }\n  }\n}"}' --compressed
```
Response:
```
{"data":{"transactions":{"edges":[]}}}
```

On the other hand - it works properly for this tx: https://viewblock.io/arweave/tx/E9wwCPtwMQk-0xQwKeZuyYhpFU8dL7pNSBYchsOMstc
- i.e. the GQL endpoint properly returns the transaction:
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/84311757/195124358-611a4c5f-9706-4b65-b5bd-4da300cd3f0e.png">

I've verified this for one of the contracts (https://sonar.warp.cc/#/app/contract/XIutiOKujGI21_ywULlBeyy-L9d8goHxt0ZyUayGaDg) and there are (at the time of writing this issue) in total **52** transactions that are not returned via GQL (from all of the **2235** interactions that where posted via Bundlr for this contract). 

The full list of currently missing transactions for this contract:
```
qeFKS_ldCiYGfdsIi3dfL_XgxOXcRzSPl26qboe4t-Q
Vw9JSfZojaDIuGRaXQ2NH1FknO8DmenlXlYhoMsWvgw
jkSwdfj0qhjpn5MLje9ZwlshV73AD6YOFT-LDkyu5As
Ff5ZtAGm4vUKFgcIZepYQ91PPNGzqS4qzCw0y71fLac
8ZN99hlt4Bm23rxCCcrSkcJzBB-3LoLszLokEQT65N8
ceOqNWANVk63VTtH4PH01cGsRDljP5c0kb39CQz_3QE
Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg
jxcA3ouhPChemHrbBU50ovX8xJIHguzj9GoAwUsri7k
NgTYuzCaDQ9gztlGSgt2XJze_vouGQVjQHTRynLpt4Y
RC2uhdZLAqGzPoQ7eugc9WCHqhLA1D36DGibqRQez9I
5p7PvYK4xzcPU_W0A1CbOhKuU9yULshFVMJHU7nSx-k
VCcVqeTuyYqQZjMRvwXgHg5y3NUdZiKMwgkPCAaDDQ4
7_W48I23zTAC9RMazRKcwPgU3SiqYj4qW96JnShfmjQ
xlRSWYNi8oGlv7c0ulkzT54zE0tXwJ0PQtGqH8DhVS0
rUwbXScTS1XmYXrvIA02ky7910veSfkUz6iFk0mibWc
uzKoghNes14cgcPZIUnoaqY0gqvXVG0sCSk7Qlz304w
DkgZl4Gv-L6OFJAGutMm4rbPLQFBClqVTFQcwZTayFg
lY2L47jNhGMTKB3Ha0FEkBdzxBFKMJXFPIOkOvHOQb0
MIhHbgQd2KUZ6rE8eLXd4taZMq1DtRuaGTYy_0Sv1j4
D6gCog2aDF9smvb8ERvQZEkTLtNbbAtifQGOy55v_z8
6hidPz5jz8ASuw4wuRjE1T0TDftO8QWVKUo3GPP3cO8
nF_w-Ndo9TT3Dyj2gyB9MiS6UuzUaXRQVKtNUFbMq54
prCjm5egyShDZValkZ9FrGpHyS8LXhGQQpPiy7JQ8hg
EWEx6ujStzW1xZilUe_trTv7wSXPxAi0h3i7C4MpV6A
N3mGGmQaor1Mk0GHmCpiZJa3tjiobWnGRJA45Cx8_f0
hbDI5Iao0qqCx27SOlQ5PR3bxkr1aWsDroWbftpxGHw
mTeOb2esaTQQiBtxsMaX0UX9Onga6T5xmJsTa3oxlfY
h6BLsxgzYLVl7ByrWzFjVyKCg_h-4pF3qpjWr4dIWHg
jSObM0jZ6tdFWfMTDw5UJfBFDTKAxc5VrY2wzcpfbmg
C-Q5EXkN5NISZq4BsIKjs41SS0uOpwKqw0QFuTGgp2Q
O9cJL7FvyX5xkm0gnECCokJyF78r0ZFmzSOYE0E6lQk
xHzciCE48TtojGMsrLPhS2ohpaJRjp9enxawkw3hBns
G7bxMpC_vNi5KtjnNkOGeRnqttWnAFyS79IpV4C_D5E
46EHsym2Cth0sSGs2k_AGFYqXvz3Kaw0aBQ2orSdHNk
tFqxytedUmhhiwZWLlBbmKgzil3ewKDQ-itJK7OEUAc
0_5lLxG1tDyvy25kTV4LpTqI0vRCDUy6bJ8JpXoa9Ck
oqFUuBK6T49VKrtXsJXxBlYp64R7meObWkcpc2v6nHI
pwREDR8XGB-IrZfhk6IUg9ZjTEfV6CLwPKc04cDFbE0
0EOKPkusmaYbvAjIFr8I885D57fsoGutV5jKE-s-R-4
TXqhRwEvhESY32mZ7hv_ICS3gGiVnHjtNuSN-_KN9h4
LB2VBIj3LiLYHCXlL6dtAMtlSjPI9a7dleGcmfnQUaY
AUTOqaPav55Kyaq_B2y0xfjCruwY_q6H_n0Y-84YSq4
wPlv3HqD65UC9zxenIwISMltjBTDR_RX6ocGizTzioM
Mz0POuaE8CcEky0N62BYB1VpPm6QuOJzaiyhiKADFKA
D-u-QiWtVjC7KvYGZpZ0u3CSrAO63qALOC4bijzcgRE
UaNabAnyEdoj6YFBkV--sSa9yA2QHBGJpjmH4_n_TgE
uEM3Et7h0eCAXf_-lgFSTUZh6KKnJVOniChaqOWtKgo
7vxyJPDGO69arjIwpeXbPV9sdsrXE4oMv-HE_J0YqcA
_rfO0uqScHcPY2pIE1z52VAPV_WCkMreSmHtRm8GIQQ
9seGQI8A6M-4cUBgeZNlNsUqOD5pXAPMtFJ5ptdhqhQ
pTZpbqsaOL_vJ72gH-N0DvjFZC55hWHpG9iV2blnSRw
h5eZlmB-saxY_TIJo4I-rRFnzrYY7XAYl5ZjNAjZgZQ
```

**NOTE:** some of the above transactions are not visible (at the time of writing) on Viewblock (probably not yet indexed), but are available via `arweave.net/{txId}`:
https://viewblock.io/arweave/tx/D-u-QiWtVjC7KvYGZpZ0u3CSrAO63qALOC4bijzcgRE
https://arweave.net/D-u-QiWtVjC7KvYGZpZ0u3CSrAO63qALOC4bijzcgRE

**NOTE 2:** It turns out that some of the transactions are not available on Viewblock, arweave.net/{txId} and GQL endpoint. An example of such transactions is `C-Q5EXkN5NISZq4BsIKjs41SS0uOpwKqw0QFuTGgp2Q`

**Comment by ppedziwiatr** [comment]
L1 tx examples (data item id -> L1 tx id):
`qeFKS_ldCiYGfdsIi3dfL_XgxOXcRzSPl26qboe4t-Q` -> `BjxJUg-zK76CZPY9L_4WABj79SSBNzTzUkv1I5exlt4`
`rUwbXScTS1XmYXrvIA02ky7910veSfkUz6iFk0mibWc` -> `S5wSnV4M4NBmAXxSayViByEOQt53ZCk_70YgnFhj4q0`
`jxcA3ouhPChemHrbBU50ovX8xJIHguzj9GoAwUsri7k` -> `BjxJUg-zK76CZPY9L_4WABj79SSBNzTzUkv1I5exlt4`

**Comment by arielmelendez** [comment]
BjxJUg-zK76CZPY9L_4WABj79SSBNzTzUkv1I5exlt4 has been reindexed by the arweave.net gateway.

**Comment by arielmelendez** [comment]
S5wSnV4M4NBmAXxSayViByEOQt53ZCk_70YgnFhj4q0 has been reindexed by the arweave.net gateway.

**Comment by ppedziwiatr** [comment]
Hey. Thanks for the fixing the S5w.
As I wrote - those were only the two _examples_. This definitely does not close the problem as a whole.

I've manually checked all the ids from this issue on ViewBlock.

More not fully unbundled L1 txs:
1. Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg -> Wd7NYKY51lqR5Bjy6BoHLkrhHYQ8LQfxvsZ9itWX_xc (https://viewblock.io/arweave/tx/Bz3escGwssOuhfEvWgIBueV-DfkYHwSJRJ5jX4YZngg)
2. uzKoghNes14cgcPZIUnoaqY0gqvXVG0sCSk7Qlz304w -> sm_rW809dglTOdGeQ0qrhMIJv0gglY6dqKaSuMRP87U
(https://viewblock.io/arweave/tx/uzKoghNes14cgcPZIUnoaqY0gqvXVG0sCSk7Qlz304w)


Unfortunatelly, I cannot verify all the transactions from this list via Viewblock - because some of them are not indexed by Viewblock (even though `arweave.net/{txId}` returns them properly).

Those transactions are:
```
8ZN99hlt4Bm23rxCCcrSkcJzBB-3LoLszLokEQT65N8
ceOqNWANVk63VTtH4PH01cGsRDljP5c0kb39CQz_3QE
5p7PvYK4xzcPU_W0A1CbOhKuU9yULshFVMJHU7nSx-k
VCcVqeTuyYqQZjMRvwXgHg5y3NUdZiKMwgkPCAaDDQ4
jSObM0jZ6tdFWfMTDw5UJfBFDTKAxc5VrY2wzcpfbmg
0EOKPkusmaYbvAjIFr8I885D57fsoGutV5jKE-s-R-4
TXqhRwEvhESY32mZ7hv_ICS3gGiVnHjtNuSN-_KN9h4
AUTOqaPav55Kyaq_B2y0xfjCruwY_q6H_n0Y-84YSq4
wPlv3HqD65UC9zxenIwISMltjBTDR_RX6ocGizTzioM
D-u-QiWtVjC7KvYGZpZ0u3CSrAO63qALOC4bijzcgRE
uEM3Et7h0eCAXf_-lgFSTUZh6KKnJVOniChaqOWtKgo
```
